### PR TITLE
Mytestcomments

### PR DESCRIPTION
**Pull request by rajeswar18**

### CODE REVIEW

**Comment by nouiz** [comment]
instead of a merge, can you do a rebase? This keep the history more clean.

File: theano/tests/test_2nd_order_grads.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
-"""
-Test for jacobian/hessian functions in Theano
-"""
+# Test for jacobian/hessian functions in Theano
```
Comment (nouiz) [inline theano/tests/test_2nd_order_grads.py]: This isn't a test, so don't change the format. Idem for all files

File: theano/tests/test_printing.py [hunk 2]
```diff
@@ -108,11 +108,11 @@ def test_min_informative_str():
...
-    reference = """A. Elemwise{add,no_inplace}
- B. C
- C. Elemwise{add,no_inplace}
-  D. D
-  E. E"""
+    reference = #A. Elemwise{add,no_inplace}
+ # B. C
+ # C. Elemwise{add,no_inplace}
+ # D. D
+ # E. E
```
Comment (nouiz) [inline theano/tests/test_printing.py]: undo. This isn't a comment, but a python string we need.

Comment (rajeswar18) [reply theano/tests/test_printing.py]: sure

**Comment by slefrancois** [comment]
There are some [flake8 errors](https://travis-ci.org/Theano/Theano/jobs/274682432#L1260).

**Comment by nouiz** [comment]
There is some travis failure related to flake8: https://travis-ci.org/Theano/Theano/jobs/274698832#L1265

You can run that test locally before pushing to make sure everything is good:

    nosetests theano/tests/test_flake8.py

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
The rebase was a mess, so we cherry-picked the main commit instead. The date is correct but the log isn't chronological.

File: theano/tests/test_flake8.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
-"""
-Test flake8 errors.
-"""
+
```
Comment (nouiz) [inline theano/tests/test_flake8.py]: At the top of files, we want docstring, not comment.

Comment (rajeswar18) [reply theano/tests/test_flake8.py]: ok

Comment (rajeswar18) [reply theano/tests/test_flake8.py]: I retained the doc strings at the top now

**Comment by slefrancois** [comment]
[Assertion error in test_printing](https://travis-ci.org/Theano/Theano/jobs/276067816#L2087), check answer format.

**Comment by slefrancois** [comment]
fixes #6238

**Comment by slefrancois** [comment]
That would mean that issue @6238 is already complete, these were the last function docstrings.

**Pull request closed without merging**